bootstrap.sh

```diff
@@ -126,43 +125,6 @@ do
 
        --functests)
             functests=ON
-            cat <<EOF
-
-==============================================================================
-        *** Use of third party software: Google Testing Framework ***
-------------------------------------------------------------------------------
-The functional test suite requires Google Testing Framework which comes with
-its own license. The license can be viewed via
-        https://github.com/google/googletest/blob/release-1.8.1/LICENSE
-Do you agree with the license [yes/no] ?
-EOF
-            read answer
-            if [ x$answer != xyes ]; then
-cat <<EOF
-------------------------------------------------------------------------------
-The answer was is '$answer'. For agreement, please type 'yes'.
-
-Agreement with Google Testing Framework license is necessary for the
-functioning of the test suite. Please do not use the --functests parameter.
-==============================================================================
-EOF
-                exit 1
-            else
-cat <<EOF
-------------------------------------------------------------------------------
-User agrees with Google Testing Framework license. It will be downloaded during
-the build.
-==============================================================================
-EOF
-                googletest_license_agreement=YES
-            fi
-
-            shift
-            ;;
-
-       --functests=i-agree-with-googletest-license)
-            functests=ON
-            googletest_license_agreement=YES
```

Comment:
Removing this makes the bootstrap script a lot simpler indeed, but I remember I made the effort because it would not be clear enough otherwise that 3rd party sources would be drawn in.

Reply:
Are you sure we need to worry about licensing? CMake have examples on very concise googletest integration -- https://cmake.org/cmake/help/latest/module/FetchContent.html#id6 -- without any mention of license issues.

Reply:
Brought it back